tests/datetime_test.py

```diff
@@ -1,10 +1,27 @@
 from common import *
 import numpy as np
 
-def test_dt():
-	t = np.array(['2007-07-13', '2006-01-13', '2010-08-13'], dtype='datetime64')
-	ds = vaex.from_arrays(t=t)
-	df = ds.to_pandas_df()
-	assert ds.t.dt.year.values.tolist() == df.t.dt.year.values.tolist()
-	assert ds.t.dt.dayofweek.values.tolist() == df.t.dt.dayofweek.values.tolist()
-	assert ds.t.dt.hour.values.tolist() == df.t.dt.hour.values.tolist()
+
+def test_datetime_operations():
+    date = [np.datetime64('2009-10-12T03:31:00'),
+            np.datetime64('2018-02-11T10:17:34'),
+            np.datetime64('2015-11-12T11:34:22'),
+            np.datetime64('2003-03-03T00:33:15'),
+            np.datetime64('2014-07-23T15:08:05'),
+            np.datetime64('2011-01-01T07:02:01')]
+
+    df = vaex.from_arrays(date=date)
+    pandas_df = df.to_pandas_df()
+
+    assert df.date.dt.hour.values.tolist() == pandas_df.date.dt.hour.values.tolist()
+    assert df.date.dt.minute.values.tolist() == pandas_df.date.dt.minute.values.tolist()
+    assert df.date.dt.second.values.tolist() == pandas_df.date.dt.second.values.tolist()
+    assert df.date.dt.day.values.tolist() == pandas_df.date.dt.day.values.tolist()
+    assert df.date.dt.day_name.values.tolist() == pandas_df.date.dt.day_name().values.tolist()
+    assert df.date.dt.month.values.tolist() == pandas_df.date.dt.month.values.tolist()
+    assert df.date.dt.month_name.values.tolist() == pandas_df.date.dt.month_name().values.tolist()
+    assert df.date.dt.year.values.tolist() == pandas_df.date.dt.year.values.tolist()
+    assert df.date.dt.is_leap_year.values.tolist() == pandas_df.date.dt.is_leap_year.values.tolist()
```

Comment:
does one of them contain a leapyear?
you could do:
assert any(df.date.dt.is_leap_year.values.tolist())

Reply:
Good point. I made one of them to contain a leap year. I still find it better to just do a direct comparison across all elements.

Reply:
I mean both. 
```python
assert df.date.dt.is_leap_year.values.tolist() == pandas_df.date.dt.is_leap_year.values.tolist()
assert any(df.date.dt.is_leap_year.values.tolist())
```

Reply:
Ok, added both.